eng/scripts/InstallVisualStudio.ps1

```diff
@@ -0,0 +1,104 @@
+<#
+.SYNOPSIS
+    Installs or updates Visual Studio on a local developer machine.
+.DESCRIPTION
+    This installs Visual Studio along with all the workloads required to contribute to this repository.
+.PARAMETER Edition
+    Selects which 'offering' of Visual Studio to install. Must be one of these values:
+        BuildTools
+        Community
+        Professional
+        Enterprise (the default)
+.PARAMETER InstallPath
+    The location on disk where Visual Studio should be installed or updated. Default path is location of latest
+    existing installation of the specified edition, if any. If that VS edition is not currently installed, default
+    path is '${env:ProgramFiles(x86)}\Microsoft Visual Studio\2019\`$Edition".
+.PARAMETER Passive
+    Run the installer without requiring interaction.
+.PARAMETER Quiet
+    Run the installer without UI and wait for installation to complete.
+.LINK
+    https://visualstudio.com
+    https://github.com/aspnet/AspNetCore/blob/master/docs/BuildFromSource.md
+.EXAMPLE
+    To install VS 2019 Enterprise, run this command in PowerShell:
+
+        .\InstallVisualStudio.ps1
+#>
+[CmdletBinding(DefaultParameterSetName = 'Default')]
+param(
+    [ValidateSet('BuildTools','Community', 'Professional', 'Enterprise')]
+    [string]$Edition = 'Enterprise',
+    [string]$InstallPath,
+    [switch]$Passive,
+    [switch]$Quiet
```

Comment:
I feel like Quiet should be the default, with an option to turn on UI so you can fiddle, rather than the other way around as it is now.

Reply:
I lined up the default (not passive, not quiet) with the VS installer. Would prefer to leave it that way as it aligns with the Principle of Least Surprise.